google/cloud/recommender/quickstart/quickstart.cc

```diff
@@ -12,23 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "google/cloud/recommender/ EDIT HERE .h"
-#include "google/cloud/project.h"
+#include "google/cloud/recommender/recommender_client.h"
 #include <iostream>
 #include <stdexcept>
 
 int main(int argc, char* argv[]) try {
-  if (argc != 2) {
-    std::cerr << "Usage: " << argv[0] << " project-id\n";
+  if (argc != 4) {
+    std::cerr << "Usage: " << argv[0]
+              << " project-id location-id recommender-id\n";
     return 1;
   }
 
   namespace recommender = ::google::cloud::recommender;
   auto client =
-      recommender::Client(recommender::MakeConnection(/* EDIT HERE */));
+      recommender::RecommenderClient(recommender::MakeRecommenderConnection());
```

Comment:
Maybe reference this:

https://cloud.google.com/recommender/docs/recommenders#recommenders

I would know where to get a project id and a location id.  Recommender IDs I had no idea bout.

Reply:
Eh. Fine. I added a comment.